samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -123,5 +123,41 @@ public async Task AssignmentsEvolvableEnumsAsync()
                 Console.WriteLine($"AssignmentsEvolvableEnumsAsync: {ex.ToString()}");
             }
         }
-    }
+
+        /// <summary>
+        /// Workflow to get assignments from all the classes which are not archived
+        /// </summary>
+        /// <param name="isTeacher">True value accepts Teacher account and false for Student account</param> 
+        public async Task CreateAndPatchAssignment(bool appOnly = false)
+        {
+            try
+            {
+                string assignmentId = string.Empty;
+
+                // Get a Graph client using delegated permissions
+                var graphClient = appOnly ? GraphClient.GetApplicationClient(_config["tenantId"], _config["appId"], _config["secret"]) : GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["teacherAccount"], _config["password"]);
+
+                // Create assignment
+                var assignment = await Assignment.CreateAsync(graphClient, _config["classId"]);
+                assignmentId = assignment.Id;
+                Console.WriteLine($"Assignment created successfully {assignment.Id} in state {assignment.Status}");
+
+                //Creating a draft assignment
+                assignment = await Assignment.PatchAsync(graphClient, _config["classId"], assignmentId);
+                
+                //Verifying whether the DisplayName parameter is updated for the draft assignment.
+                assignment = await Assignment.GetAssignmentAsync(graphClient, _config["classId"], assignmentId);
+                if(assignment.DisplayName.Contains("updated"))
+                {
+                Console.WriteLine($"DisplayName updated successfully {assignment.Id} DisplayName {assignment.DisplayName}");
+                }
+                await Assignment.DeleteAsync(graphClient, _config["classId"], assignmentId);
+
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"AssignmentsEvolvableEnumsAsync: {ex.ToString()}");
```

Comment:
CreateAndPatchAssignmentAsync

Reply:
updated to "CreateAndPatchAssignmentAsync"